From 103f1f6b4e4cfd007375cd127b8e69cd102d4097 Mon Sep 17 00:00:00 2001
From: Jan Vcelak <jvcelak@redhat.com>
Date: Tue, 1 Jan 2013 15:32:01 +0100
Subject: [PATCH] missing groff-x11 info message when gxditview not found

Adds info message that 'groff-x11' package might be missing when
executing 'groff -X' and 'gxditview' is not found.

Resolves: #530788
Signed-off-by: Jan Vcelak <jvcelak@redhat.com>
---
 src/roff/groff/pipeline.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/roff/groff/pipeline.c b/src/roff/groff/pipeline.c
index cea7593..fce0abc 100644
--- a/src/roff/groff/pipeline.c
+++ b/src/roff/groff/pipeline.c
@@ -486,6 +486,8 @@ int run_pipeline(int ncommands, char ***commands, int no_pipe)
       execvp(commands[i][0], commands[i]);
       error("couldn't exec %1: %2",
 	    commands[i][0], strerror(errno), (char *)0);
+	  if (strcmp(commands[i][0], "gxditview") == 0)
+	    fprintf(stderr, "You might be missing 'groff-x11' package.\n");
       fflush(stderr);			/* just in case error() doesn't */
       _exit(EXEC_FAILED_EXIT_STATUS);
     }
-- 
1.8.0.2

